tasks2.js

```diff
@@ -0,0 +1,184 @@
+    let sentenceTwo = 'I hope this course is not full of jargon'
+
+    console.log(sentenceTwo.includes('jargon'));
+
+// 9. Generate a random number between 0 and 100 inclusively.
+    let randomNumber = Math.random(0-100)
+    let numBtnZeroAndHundred = randomNumber*99
+    console.log(numBtnZeroAndHundred);
+
+// 10. Generate a random number between 50 and 100 inclusively.
+function getRandomNumberFifty() {
+    return Math.floor(Math.random() * 101);
+}
+
+const randomNumber5 = getRandomNumberFifty();
+console.log(randomNumber5);
+
+// 11. Generate a random number between 0 and 255 inclusively.
+    function getRandomNumber() {
+    return Math.floor(Math.random() * 256);
+}
+
+const randomNum = getRandomNumber();
+console.log(randomNum);
+
+
+/* 13. Use console.log() and escape characters to print the following pattern.
+
+1 1 1 1 1
+2 1 2 4 8
+3 1 3 9 27
+4 1 4 16 64
+5 1 5 25 125*/
+
+
+// 14. Use substr to slice out the phrase because because because from the following sentence:'You cannot end a sentence with because because because is a conjunction'
+    console.log(rule.substr(31,24))
+
+/* Exercises: Level 3 */
+//1. 'Love is the best thing in this world. Some found their love and some are still looking for their love.' Count the number of word love in this sentence.
+    const loveTwo = 'Love is the best thing in this world. Some found their love and some are still looking for their love.';
+    const loveCount = (loveTwo.match(/love/gi) || []).length; // Use /gi to match case-insensitively
+    console.log(loveCount); // Output: 3
+
+    
+//2. Use match() to count the number of all because in the following sentence:'You cannot end a sentence with because because because is a conjunction'
+    const sentenceBecause = 'You cannot end a sentence with because because because is a conjunction';
+    const becauseCount = (sentenceBecause.match(/because/gi) || []).length; // Use /gi to match case-insensitively
+    console.log(becauseCount); // Output: 3
+    
+//3. Clean the following text and find the most frequent word (hint, use replace and regular expressions).
+
+    const sentence = '%I $am@% a %tea@cher%, &and& I lo%#ve %te@a@ching%;. The@re $is no@th@ing; &as& mo@re rewarding as educa@ting &and& @emp%o@weri@ng peo@ple. ;I found tea@ching m%o@re interesting tha@n any ot#her %jo@bs. %Do@es thi%s mo@tiv#ate yo@u to be a tea@cher!? %Th#is 30#Days&OfJavaScript &is al@so $the $resu@lt of &love& of tea&ching'
+
+    // 4. Calculate the total annual income of the person by extracting the numbers from the following text. 'He earns 5000 euro from salary per month, 10000 euro annual bonus, 15000 euro online courses per month.'
+
+    const incomeText = 'He earns 5000 euro from salary per month, 10000 euro annual bonus, 15000 euro online courses per month.';
+const monthlyIncome = incomeText.match(/\d+/g).map(Number);
+
+// Calculate total annual income
+const totalAnnualIncome = (monthlyIncome[0] * 12) + monthlyIncome[1]; 
```

Comment:
`totalAnnualIncome` doesn't include the income from online courses.

Reply:
I will try it one more time